website/docs/components/page-header/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,238 @@
+
+![Page Header primary dropdown](/assets/components/page-header/page-header-actions-do-primary-dropdown.png =853x*)
+
+!!!
+
+!!! Dont
+
+Don’t use a `primary` color [Dropdown](/components/dropdown) when combining different, unrelated actions. Instead, use the `secondary` color.
+
+![Page Header primary dropdown](/assets/components/page-header/page-header-actions-dont-primary-dropdown.png =853x*)
+
+!!!
+
+### Action color and pairing
+
+While the Page Header supports up to three actions in Figma, we recommend limiting the number of actions to two for most pages. These actions should generally consist of a primary action (e.g. "Create" or "Deploy") and a secondary action (e.g., "Manage").
+
+For more details on how to combine, order, and organize buttons, refer to the [Button order, organization, and alignment](/patterns/button-organization) pattern documentation.
+
+!!! Dont
+
+Don’t pair two primary actions in a Page Header. Instead, communicate the highest priority action for the user with a primary Button or Dropdown and other actions with a secondary Button or Dropdown.
+
+![Page Header two primary actions](/assets/components/page-header/page-header-dont-two-actions.png =700x*)
+
+!!!
+
+## Size
+
+Size only pertains to the Figma component and accounts for smaller viewports by stacking elements. The Ember component supports a basic level of fluidity and responsiveness out of the box.
+
+### Large
+
+![Large Page Header variant](/assets/components/page-header/page-header-size-large.png =945x*)
+
+### Small
+
+![Small Page Header variant](/assets/components/page-header/page-header-size-small.png =300x*)
+
+### Responsiveness
+
+The Page Header component in Figma supports two sizes; `large` which accounts for the majority of desktop sizes and large tablets, and `small` which accounts for smaller tablet and mobile devices. The core difference between each variant is the vertical stacking of elements.
+
+The Ember component uses a variety of different methods to ensure fluidity and responsiveness:
+
+- By default, the component will fill the page layout it is used within. It does not have any padding or margin explicitly applied, allowing it to adapt to different layout and spacing methods.
+- The component has breakpoints by means of container queries (at `768px` and one at `400px`) that account for the majority of content within the component and stack elements in a single column as the container shrinks.
+- Elements displayed inline with each other (e.g., title and badges) have `flex-wrap: wrap;` set to wrap elements when the available space is reduced.
+
+!!! Info
+
+Additional responsive characteristics are the responsibility of the consumer and dependent on the layout and spacing methods defined at the application level.
+
+!!!
+
+## Related
```

Comment:
should we add "Button", "Breadcrumb", "Dropdown" too?

Reply:
If I remember correctly the blueprint inserts a comment recommending to only list two related components (I'm not sure why). In this case it's probably useful to capture the further nested components.